Project.js

```diff
@@ -9,9 +9,12 @@ import {
   View
 } from 'react-native';
 import EStyleSheet from 'react-native-extended-stylesheet'
+import GoogleAnalytics from 'react-native-google-analytics-bridge'
 
 class Project extends Component {
   handleClick() {
+    GoogleAnalytics.trackEvent('View Project', this.props.project.display_name);
```

Comment:
Should we make this camel case? Might want to check out how we currently track on GA to stay consistent.